components/org.wso2.carbon.identity.auth.service/src/main/java/org/wso2/carbon/identity/auth/service/AuthenticationManager.java

```diff
@@ -154,6 +156,10 @@ private List<AuthenticationHandler> filterAuthenticationHandlers(AuthenticationC
     private boolean isHandlerAllowedForResource(List<String> allowedAuthenticationHandlersForResource,
                                                 AuthenticationHandler handler) {
 
+        if (Boolean.parseBoolean(IdentityUtil.getProperty(Constants.DISABLE_BASIC_AUTH_HANDLER_CONFIG)) &&
```

Comment:
shall we rename the config as `ENABLE_BASIC_AUTH_HANDLER_CONFIG`
by default = false. If some one wants to enable set true.
Otherwise the config reading is a double negation